src/services/ui/src/main.tsx

```diff
@@ -33,12 +33,13 @@ const initializeLaunchDarkly = async () => {
       <QueryClientProvider client={queryClient}>
         <UserContextProvider>
           <LDProvider>
+            <TimeoutModal />
```

Comment:
Is there anyway to add `TimeoutModal` to protected routes so we don't have to track idleness on the login page? It'll eliminate checking `user?.user` as well